legion/engine/physics/data/physics_component_data.hpp

```diff
@@ -10,33 +10,41 @@ namespace legion::physics
 
     class PhysicsComponentData
     {
+        typedef delegate<void* (const std::vector<math::vec3>& vertices)> GenerateConvexDelegate;
```

Comment:
typedef is a little outdated, and is still from C. the modern C++ type alias is: `using Alias = OtherType;`